setup.py

```diff
@@ -66,7 +66,8 @@
             "file_contents = sceptre.resolvers.file_contents:FileContents",
             "stack_output = sceptre.resolvers.stack_output:StackOutput",
             "stack_output_external ="
-            "sceptre.resolvers.stack_output:StackOutputExternal"
+            "sceptre.resolvers.stack_output:StackOutputExternal",
+            "s3 = sceptre.resolvers.s3:S3"
```

Comment:
Okay, I can set it up in a separate repo. However, I would argue that this is not a 3rd party plugin. Just like `!stack_output` and `!file_contents`, the `template_path` resolvers are part of the core functionality of Sceptre IMHO. If people are also working on moving those resolvers to separate repos, then I agree. Otherwise I would opt for keeping a minimum set of `template_path` resolvers in the repo, to support a basic feature set.

Reply:
sorry, url reference in my previous comment was incorrect.  i have updated to the correct URL.

We should get confirmation but I'm just relaying what @ngfgrant said from a  previous discussion about plugins.. https://github.com/Sceptre/sceptre/issues/680#issuecomment-511311037